examples/models/maps.py

```diff
@@ -16,7 +16,15 @@
 [{"featureType":"administrative","elementType":"all","stylers":[{"visibility":"on"},{"lightness":33}]},{"featureType":"landscape","elementType":"all","stylers":[{"color":"#f2e5d4"}]},{"featureType":"poi.park","elementType":"geometry","stylers":[{"color":"#c5dac6"}]},{"featureType":"poi.park","elementType":"labels","stylers":[{"visibility":"on"},{"lightness":20}]},{"featureType":"road","elementType":"all","stylers":[{"lightness":20}]},{"featureType":"road.highway","elementType":"geometry","stylers":[{"color":"#c5c6c6"}]},{"featureType":"road.arterial","elementType":"geometry","stylers":[{"color":"#e4d7c6"}]},{"featureType":"road.local","elementType":"geometry","stylers":[{"color":"#fbfaf7"}]},{"featureType":"water","elementType":"all","stylers":[{"visibility":"on"},{"color":"#acbcc9"}]}]
 """)
 
-plot = GMapPlot( x_range=x_range, y_range=y_range, map_options=map_options)
+# Google Maps now requires an API key. You can find out how to get one here:
+# https://developers.google.com/maps/documentation/javascript/get-api-key
+API_KEY = "XXXXXXXXXXX"
```

Comment:
I wouldn't hold up this PR for this, but assuming we eventually make `embed.file_html` plots validate the same way as `show` does, this would circumvent the error check as it is currently implemented (it checks for `api_key is None`)